meta-integrity/classes/sign_rpm_ext.bbclass

```diff
@@ -10,6 +10,9 @@ RPM_FSK_PASSWORD ?= "password"
 
 inherit sign_rpm user-key-store
 
+GPG_DEP = "${@'' if (d.getVar('GPG_BIN', True) or '').startswith('/') else 'gnupg-native pinentry-native'}"
```

Comment:
Not sure why using startswith('/') to estimate whether GPG_BIN is set or not.

Reply:
user maybe set GPG_BIN in local.conf, it need to be a absolute path.  here just to do a simple check if it is a valid path.  But even startwith "/" not mean it is a valid gpg path,  it will fail when running the cmd.

Reply:
> user maybe set GPG_BIN in local.conf, it need to be a absolute path. here just to do a simple check if it is a valid path. But even startwith "/" not mean it is a valid gpg path, it will fail when running the cmd.

Specifying GPG_BIN in form of relative path is not working in yocto?

Reply:
relative path works, but original idea is that relative path could be confusing.   for example,
local.conf is under build/conf/local.conf,  but actually,  the cmd  runned under dir build. User maybe confuse about relative to which path?  so suppose it is a absolute path here.

but indeed,  startswith here is not very necessary, because if the path is invalid,  it still will fail during run the command. add it here just for exclude some wrong path.  I can remove this if you think it is better.

Reply:
> relative path works, but original idea is that relative path could be confusing. for example,
> local.conf is under build/conf/local.conf, but actually, the cmd runned under dir build. User maybe confuse about relative to which path? so suppose it is a absolute path here.
> 
> but indeed, startswith here is not very necessary, because if the path is invalid, it still will fail during run the command. add it here just for exclude some wrong path. I can remove this if you think it is better.

Please remove the startswith check. It is not our responsibility to sanity check everything working as expected.

Reply:
corrected.